date_calculator.py

```diff
@@ -0,0 +1,40 @@
+---------------------------Code Review Example---------------------------
+LEAST_FAVORITE_YEAR = 2020
+
+def suPerScr_etFunCtIon(month, day, year):
+    """
+    returns the day of year given a date
+    """
```

Comment:
Way too many if statements! Just make an array holding # days in each month and add based on index:

month_array=np.array([31, 28, 31.....])
days=np.sum(month_array[:month])